lisa/tools/dhclient.py

```diff
@@ -52,13 +52,15 @@ def get_timeout(self) -> int:
             paths_to_check = [
                 f"/etc/dhcp/{self._command}.conf",
                 f"/etc/{self._command}.conf",
+                f"/usr/lib/dracut/modules.d/40network/{self._command}.conf",
+                f"/usr/lib/dracut/modules.d/35network-legacy/{self._command}.conf",
             ]
 
             ls = self.node.tools[Ls]
             config_path = next(
                 (path for path in paths_to_check if ls.path_exists(path, sudo=True)), ""
             )
-
+            self._log.info(f"dhclient config paths: {paths_to_check}")
```

Comment:
no need this log, ls will print the log

Reply:
Oh thank you I forgot to remove this. It is gone now.